### PR TITLE
feat(docs): Repay Advance from Salary Slip

### PR DESCRIPTION
**Pull request by Anurag810**
image

### CODE REVIEW

**Comment by Anurag810** [comment]
https://github.com/frappe/erpnext/pull/20366

File: erpnext_com/www/docs/user/manual/en/human-resources/employee-advance.md [hunk 1]
```diff
@@ -36,7 +36,14 @@ After saving Employee Advance, Employee should [Assign document to Approver](/do
 
 After verification, the Expense Approver can Submit (Accept) the Employee Advance form or Reject the request.
 
-### 3.2 Make Payment Entry
+### 3.2 Repay Employee Advance From salary
+
+This feature allows to pay the Employee Advance Unclaimed Amount via salary.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/human-resources/employee-advance.md]:
```suggestion
This feature allows paying the Employee Advance Unclaimed Amount via salary.
```

File: erpnext_com/www/docs/user/manual/en/human-resources/employee-advance.md [hunk 2]
```diff
@@ -36,7 +36,14 @@ After saving Employee Advance, Employee should [Assign document to Approver](/do
...
+If 'Repay unclaimed amount from salary' is checked then you need to fill Payroll Date and Salary Component. The Unclaimed amount will be deducted from salary if payroll date is in the interval when Salary is processed, It will be added to the deduction.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/human-resources/employee-advance.md]: sentence is unclear

**Comment by prasadarr** [comment]
docs have been moved
https://github.com/frappe/erpnext_documentation

**Pull request closed without merging**